src/main/resources/AntwoordenModel_v2_2_2.csv

```diff
@@ -53,8 +53,8 @@ Ja,Ja,Nee,Nee,,Ja,,,,,,Nee,2,,Ja,,,14,G,Nee,Nee,Nee,Nee,1450,Ingezeten - minderj
 Ja,Ja,Nee,Ja,Ja,Ja,,,,,,Nee,2,,Ja,,,14a,G,Nee,Nee,Nee,Nee,1451,Ingezeten - minderjarig - adoptie NL akte - categorie 11 ouder2 - beide ouders onbevoegd - soort gezag is G - tijdelijk geen gezag
 Ja,Ja,Nee,Nee,,Ja,,,,,,Ja,,,,,,15e,N,Nee,Nee,Nee,Nee,1500,gezag niet bepalen omdat relevante gevens ontbreken bij het bepalen van het aantal ouders. Het gaat om de volgende gegevens: %s
 Ja,Ja,Nee,Ja,Ja,Ja,,,,,,Ja,,,,,,15ae,N,Nee,Nee,Nee,Nee,1501,gezag niet bepalen omdat relevante gevens ontbreken bij het bepalen van het aantal ouders. Het gaat om de volgende gegevens: %s
-Ja,Ja,Nee,Nee,,Ja,Geen_ouders,,,,,Ja,,,,,,15,G,Nee,Nee,Nee,Nee,1550,Ingezeten - minderjarig - categorie 11 en recentere gebeurtenis(sen) - heeft geen ouders - soort gezag is G - tijdelijk geen gezag
-Ja,Ja,Nee,Ja,Ja,Ja,Geen_ouders,,,,,Ja,,,,,,15a,G,Nee,Nee,Nee,Nee,1551,Ingezeten - minderjarig - adoptie NL akte - categorie 11 en recentere gebeurtenis(sen) - heeft geen ouders - soort gezag is G - tijdelijk geen gezag
+Ja,Ja,Nee,Nee,,Ja,Geen_ouders,,,,,Ja,,,,,,15,G,Nee,Nee,Nee,Nee,1550,Ingezeten - minderjarig - categorie 11 en recentere gebeurtenissen) - heeft geen ouders - soort gezag is G - tijdelijk geen gezag
```

Comment:
Ik zou verwachten dat zowel haakje openen als sluiten verdwijnen als we besluiten om haakjes weg te werken.

:heavy_check_mark: gebeurtenis(sen) => gebeurtenissen
:question:  gebeurtenis(sen) => gebeurtenissen)